src/api/store/message.py

```diff
@@ -38,16 +42,55 @@ def get_logo(id):
         if com:
             return com.logo.file.url if com.logo else None
         
-    
 
-
-def get_message_recipients(audience, audience_type):
-    if not is_null(audience):
+def get_message_recipients(audience, audience_type, community_ids,sub_audience_type):
+    
+    if is_null(audience): return None
+    if community_ids and not isinstance(community_ids, list):
+        community_ids = community_ids.split(",")
+
+    if audience_type == "COMMUNITY_CONTACTS":
```

Comment:
define this as some enum somewhere so we dont mistakenly mess up the spelling sometime.  

eg.
```
class AudienceType(Enum):
    COMMUNITY_CONTACTS = "COMMUNITY_CONTACTS"
    CADMIN = "CADMIN"
    
 then somewhere in here you can do like
 audience_type = AudienceType(audience_type)
 if audience_type == AudienceType.COMMUNITY_CONTACTS:
     # do something magical
 elif audience_type == AudienceType.CADMIN:
     # do some other magical thing
```